### PR TITLE
Fix docblocks

### PR DESCRIPTION
**Pull request by mikeSimonson**

### CODE REVIEW

File: lib/Doctrine/DBAL/Migrations/Configuration/Configuration.php [hunk 1]
```diff
@@ -638,7 +638,9 @@ public function getNextVersion()
     /**
      * Returns the version with the specified offset to the specified version.
      *
-     * @return string|null A version string, or null if the specified version
+     * @param $version string
```
Comment (alcaeus) [inline lib/Doctrine/DBAL/Migrations/Configuration/Configuration.php]: The type precedes the variable name. This goes for all param documentation in this PR.

File: lib/Doctrine/DBAL/Migrations/FileQueryWriter.php [hunk 2]
```diff
@@ -51,6 +51,10 @@ public function __construct(string $columnName, string $tableName, ?OutputWriter
 
     /**
      * TODO: move SqlFileWriter's behaviour to this class - and kill it with fire (on the next major release)
+     * @param string $path
```
Comment (alcaeus) [inline lib/Doctrine/DBAL/Migrations/FileQueryWriter.php]: You might as well omit these as the method signature already contains type information.

File: lib/Doctrine/DBAL/Migrations/QueryWriter.php [hunk 3]
```diff
@@ -28,7 +28,10 @@
 interface QueryWriter
 {
     /**
-     * @throws \Doctrine\DBAL\Exception\InvalidArgumentException
+     * @param string $path
```
Comment (alcaeus) [inline lib/Doctrine/DBAL/Migrations/QueryWriter.php]: Can be omitted

File: lib/Doctrine/DBAL/Migrations/Tools/Console/Helper/MigrationStatusInfosHelper.php [hunk 4]
```diff
@@ -77,7 +77,9 @@ private function getFormattedVersionAlias($alias)
         return $this->configuration->getDateTime($version) . ' (<comment>' . $version . '</comment>)';
     }
 
-    /** @var Version[] */
+    /**
+     * @return array|Version[]
```
Comment (alcaeus) [inline lib/Doctrine/DBAL/Migrations/Tools/Console/Helper/MigrationStatusInfosHelper.php]: The array return type is useless when you add Version[]

File: lib/Doctrine/DBAL/Migrations/Configuration/Configuration.php (hunk 1 above)
Comment (mikeSimonson) [reply lib/Doctrine/DBAL/Migrations/Configuration/Configuration.php]: Weird are you sure ? Because most of those were auto generated by PHPStorm.

Comment (alcaeus) [reply lib/Doctrine/DBAL/Migrations/Configuration/Configuration.php]: See other examples in the same file.

**Pull request merged**